prov/efa/src/rdm/efa_rdm_pke.h

```diff
@@ -219,17 +219,14 @@ struct efa_rdm_pke {
 	 */
 	struct rxr_pkt_sendv *send;
 
-	/** @brief Work request struct used by rdma-core.
-	 *  @todo move this field out of efa_rdm_pke, which requires re-implement the buld send.
-	 */
-	struct efa_send_wr *send_wr;
-
 	/**
 	 * @brief Work request struct used by rdma-core for receive.
 	 * @todo move this field out of efa_rdm_pke to a separate pool.
 	 */
 	struct efa_recv_wr recv_wr;
 
+	uint8_t pad2[8];
+
```

Comment:
A comment to why we need this padding and how we know it should be 8 bytes?

Reply:
There is a static assertion in the code to make sure the struct is 128 bytes. If pad size is not right, compilation would fail.